restapi/src/main/java/io/stargate/web/docsapi/service/DocsApiConfiguration.java

```diff
@@ -1,41 +1,42 @@
 package io.stargate.web.docsapi.service;
 
-import io.stargate.web.docsapi.dao.DocumentDB;
-
 /** The configuration for the document API. */
 public interface DocsApiConfiguration {
 
   /** Default config instance */
   DocsApiConfiguration DEFAULT = new DocsApiConfiguration() {};
 
   /** @return Returns max allowed document page size that user can request. */
-  default int getMaxPageSize() {
-    return DocumentDB.MAX_PAGE_SIZE;
-  }
+  int MAX_PAGE_SIZE = 20;
 
   /**
    * @return Returns approximate storage page size to use when querying database, based on the
    *     amount of documents we are searching for. We consider that in average documents have 16
    *     fields.
    */
   default int getApproximateStoragePageSize(int numberOfDocuments) {
-    return Math.min(numberOfDocuments * 16, DocumentDB.MAX_STORAGE_PAGE_SIZE);
+    return Math.min(numberOfDocuments * 16, getMaxStoragePageSize());
   }
 
   /**
    * @return The maximum storage page size that should be allowed when doing any doc API queries.
    */
   default int getMaxStoragePageSize() {
-    return DocumentDB.MAX_STORAGE_PAGE_SIZE;
+    return Integer.getInteger("stargate.document_search_page_size", 1000);
```

Comment:
should we extract those to the fields and return the field here? this way we are always parsing the strings, but sure why would we have that overhead?

Also fine to move to constants or some properties class